tox.ini

```diff
@@ -7,6 +7,7 @@ setenv =
     LC_ALL=C
 deps =
     freezegun
+;    PyGObject
```

Comment:
No?  Why?  If we want pygobject in tox, we can remove `skip_install = true` and `pip install --no-deps -e .`, and tox will discover that pygobject is needed because it's listed in install_requires.

Reply:
Humm ok. I didn't know that (not enough used to tox btw ^^)

Reply:
Fixed